src/backend/marsha/development/management/commands/load_development_datasets.py

```diff
@@ -0,0 +1,298 @@
+            consumer_site=localhost_cs,
+            organization=organization,
+            created_by=self.superuser,
+            lti_id="course-v1:playlist+created+by+admin",
+            is_portable_to_playlist=False,
+        )
+        if created:
+            VideoFactory.create_batch(3, playlist=playlist_admin)
+            DocumentFactory.create_batch(3, playlist=playlist_admin)
+            MarkdownDocumentFactory.create_batch(3, playlist=playlist_admin)
+
+        playlist_admin_permission, created = Playlist.objects.get_or_create(
+            title="admin user playlist",
+            consumer_site=localhost_cs,
+            organization=organization,
+            created_by=self.superuser,
+            lti_id="course-v1:playlist+for_admin_user",
+            is_portable_to_playlist=False,
+        )
+        if created:
+            PlaylistAccess.objects.get_or_create(
+                user=self.superuser,
+                playlist=playlist_admin_permission,
+                role=ADMINISTRATOR,
+            )
+            VideoFactory.create_batch(3, playlist=playlist_admin_permission)
+            DocumentFactory.create_batch(3, playlist=playlist_admin_permission)
+            MarkdownDocumentFactory.create_batch(3, playlist=playlist_admin_permission)
+            # also add random playlists
+            PlaylistAccessFactory.create_batch(
+                5,
+                user=self.superuser,
+                role=ADMINISTRATOR,
+            )
+
+        playlist_instructor, created = Playlist.objects.get_or_create(
+            title="Playlist created by instructor",
+            consumer_site=localhost_cs,
+            organization=organization,
+            created_by=self.instructor,
+            lti_id="course-v1:playlist+created+by+instructor",
+            is_portable_to_playlist=False,
+        )
+        if created:
+            VideoFactory.create_batch(3, playlist=playlist_instructor)
+            DocumentFactory.create_batch(3, playlist=playlist_instructor)
+            MarkdownDocumentFactory.create_batch(3, playlist=playlist_admin)
+
+        playlist, created = Playlist.objects.get_or_create(
+            title="Playlist created by unknown user",
+            consumer_site=localhost_cs,
+            organization=organization,
+            lti_id="course-v1:playlist+with+direct+access",
+            is_portable_to_playlist=False,
+        )
+        PlaylistAccess.objects.get_or_create(
+            user=self.playlist_admin,
+            playlist=playlist,
+            role=ADMINISTRATOR,
+        )
+        PlaylistAccess.objects.get_or_create(
+            user=self.playlist_instructor,
+            playlist=playlist,
+            role=INSTRUCTOR,
+        )
+        PlaylistAccess.objects.get_or_create(
+            user=self.playlist_student,
+            playlist=playlist,
+            role=STUDENT,
+        )
+
+        if created:
+            VideoFactory.create_batch(3, playlist=playlist)
```

Comment:
Can you also create 3 webinars ? `create_batch(3, playlist=playlist, live_state=IDLE, live_type=JITST, upload_state=PENDING)` should do the job